svelte-cloudinary/src/lib/cloudinary.ts

```diff
@@ -25,4 +25,10 @@ export async function pollForProcessingImage(options: PollForProcessingImageOpti
     return false;
   }
   return true;
+}
+
+export function checkCloudinaryCloudName(cloudName: string) {
+  if (!cloudName) {
+    throw new Error("A Cloudinary Cloud name is required, please make sure VITE_PUBLIC_CLOUDINARY_CLOUD_NAME is set and configured in your environment");
```

Comment:
Last change request.
Can you add a prefix to this error? Something like `[Svelte-cloudinary]`

This is how the error looks 
![CleanShot 2023-10-13 at 08 08 36](https://github.com/cloudinary-community/svelte-cloudinary/assets/282006/35322856-08e4-43d4-ada2-bbf0a1c14df5)

And I think it would  be nice to quickly identify from where it comes